torch/csrc/jit/codegen/cuda/type.cpp

```diff
@@ -319,6 +319,22 @@ TORCH_CUDA_API c10::optional<std::string> cast_func_str(
   }
 }
 
+size_t dataTypeSize(DataType type) {
+  switch (type) {
+    // TODO: Is Bool actually supported? If so, is it one byte?
```

Comment:
I don't think bool or int tensors are supported, but we do have bool and int scalar types for obvious reasons. @kevinstephano to confirm

Reply:
OK. What would you say for the size of `DataType::Bool`? I feel just returning `sizeof(bool)` seems reasonable. It seems that's what `c10::elementSize` does.